testing/kuttl/e2e/cluster-migrate/00--non-crunchy-cluster.yaml

```diff
@@ -0,0 +1,195 @@
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      postgres-operator-test: kuttl
+      app.kubernetes.io/name: postgresql
+      app.kubernetes.io/instance: non-crunchy-cluster
+      app.kubernetes.io/component: primary
+  template:
+    metadata:
+      name: non-crunchy-cluster
+      labels:
+        postgres-operator-test: kuttl
+        app.kubernetes.io/name: postgresql
+        app.kubernetes.io/instance: non-crunchy-cluster
+        app.kubernetes.io/component: primary
+    spec:
+      serviceAccountName: default
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - podAffinityTerm:
+                labelSelector:
+                  matchLabels:
+                    postgres-operator-test: kuttl
+                    app.kubernetes.io/name: postgresql
+                    app.kubernetes.io/instance: non-crunchy-cluster
+                    app.kubernetes.io/component: primary
+                namespaces:
+                  - "default"
+                topologyKey: kubernetes.io/hostname
+              weight: 1
+      securityContext:
+        fsGroup: 1001
+      hostNetwork: false
+      hostIPC: false
+      containers:
+        - name: postgresql
+          image: docker.io/bitnami/postgresql:14.5.0-debian-11-r37
+          imagePullPolicy: "IfNotPresent"
+          securityContext:
+            runAsUser: 1001
+          env:
+            - name: BITNAMI_DEBUG
+              value: "false"
+            - name: POSTGRESQL_PORT_NUMBER
+              value: "5432"
+            - name: POSTGRESQL_VOLUME_DIR
+              value: "/bitnami/postgresql"
+            - name: PGDATA
+              value: "/bitnami/postgresql/data"
+            - name: POSTGRES_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: non-crunchy-cluster
+                  key: postgres-password
+            - name: POSTGRESQL_ENABLE_LDAP
+              value: "no"
+            - name: POSTGRESQL_ENABLE_TLS
+              value: "no"
+            - name: POSTGRESQL_LOG_HOSTNAME
+              value: "false"
+            - name: POSTGRESQL_LOG_CONNECTIONS
+              value: "false"
+            - name: POSTGRESQL_LOG_DISCONNECTIONS
+              value: "false"
+            - name: POSTGRESQL_PGAUDIT_LOG_CATALOG
+              value: "off"
+            - name: POSTGRESQL_CLIENT_MIN_MESSAGES
+              value: "error"
+            - name: POSTGRESQL_SHARED_PRELOAD_LIBRARIES
+              value: "pgaudit"
+          ports:
+            - name: tcp-postgresql
+              containerPort: 5432
+          livenessProbe:
+            failureThreshold: 6
+            initialDelaySeconds: 30
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 5
+            exec:
+              command:
+                - /bin/sh
+                - -c
+                - exec pg_isready -U "postgres" -h 127.0.0.1 -p 5432
```

Comment:
⛏️ Can we change `127.0.0.1` to `localhost` here?

🌱 This IPv4 address (probably?) won't work in an IPv6-only environment.

Same a few lines down.